src/frontends/paddle/src/op/meshgrid.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs meshgrid(const NodeContext& node) {
+    auto inputs = node.get_ng_inputs("X");
+    OutputVector dims;
+    for (const auto& input : inputs) {
```

Comment:
any restriction to the number of inputs? could there be a test case with more than 2 inputs?

Reply:
Adjust the testcase to test 3 inputs. There is no explicate limit.